PocketKit/Sources/PocketKit/PremiumUpgrade/Model/SubscriptionStore.swift

```diff
@@ -29,6 +29,7 @@ public enum PurchaseState {
 }
 
 /// Generic type representing a subscription store
+@MainActor
```

Comment:
should this be a main actor if it handles processing?

Reply:
It should be fine since the processing is done by the system anyway (purchases). That said, we should look into the new views that `StoreKit` makes available for `SwifUI`, in a future evolution of our purchase interface.